src/seedlink/plugins/serial_plugin/serial_plugin.cc

```diff
@@ -218,7 +220,8 @@ class SEEDLog
       }
   };
 
-#if !defined(__GNU_LIBRARY__) && !defined(__GLIBC__)
+#if !defined(__GNU_LIBRARY__) && !defined(__GLIBC__) 
```

Comment:
I think this guard is not required anymore. If we found cfmakeraw then use it otherwise use our function.